awx/ui_next/src/screens/Job/JobTypeRedirect.jsx

```diff
@@ -24,29 +30,49 @@ class JobTypeRedirect extends Component {
 
   async loadJob() {
     const { id } = this.props;
+    this.setState({ isLoading: true });
     try {
       const { data } = await UnifiedJobsAPI.read({ id });
+      const job = data.results[0];
+      this.setState({
+        job,
+        isLoading: false,
+        error: job ? null : NOT_FOUND,
```

Comment:
Do we sometimes receive `null` for jobs we request from the API? I'm trying to think of a way to reproduce this

Reply:
edit: nevermind, this is a list request and `null` is the default. got it :+1: